be/src/exprs/encryption_functions.cpp

```diff
@@ -418,4 +419,136 @@ Status EncryptionFunctions::sha2_close(FunctionContext* context, FunctionContext
     return Status::OK();
 }
 
+StatusOr<ColumnPtr> EncryptionFunctions::fpe_encrypt(FunctionContext* ctx, const Columns& columns) {
+    auto src_viewer = ColumnViewer<TYPE_VARCHAR>(columns[0]);
+    auto key_viewer = ColumnViewer<TYPE_VARCHAR>(columns[1]);
+
+    auto size = columns[0]->size();
+    ColumnBuilder<TYPE_VARCHAR> result(size);
+    Status status;
+    std::string value;
+
+    for (int row = 0; row < size; ++row) {
+        if (src_viewer.is_null(row)) {
+            result.append_null();
+            continue;
+        }
+        auto src_value = src_viewer.value(row);
+        auto key_value = key_viewer.value(row);
+        auto key_size = key_value.size;
+        if (key_size != 16 && key_size != 24 && key_size != 32) {
+            throw std::runtime_error("key size must 16 or 24 or 32");
```

Comment:
use StatusOr::xxxError instead of throw std::runtime_error

Reply:
Replace with Status::InternalError

Reply:
Replace with Status::InternalError

Reply:
Replaced by Status::InvalidArgument